### ISSUE TITLE
[Profile] GDPR Privacy by Default

### ISSUE THREAD

**Issue by datenbrei** [Human-Connection/Human-Connection#286]
As a user of the network
I want to decide which information, like my recommendations or my solved CanDos or my badges or other information, is shown to the public or to my friends or just to me
in order to keep my privacy and also to follow the privacy by design demand of the European Unions GDPR article 25: https://gdpr-info.eu/art-25-gdpr/

**Comment by mattwr18** [comment]
I agree this is important, but maybe it could be broken into smaller issues? we could create an issue for each of the bits of info we want to make private? Like at the moment, we have given users the ability to make their shouts public or not... allow embed iframes, as well @datenbrei

**Comment by datenbrei** [comment]
Yes. I'd say, this should be an EPIC, so that smaller decisions or settings can be done. I will extend this one.

**Comment by datenbrei** [comment]
After discussion with Dennis I took out the registration date to be unvisible.

**Comment by sushidave** [comment]
For posts this is related to issue #2198 .
